### ISSUE TITLE
poor training performance with DELG

### ISSUE THREAD

**Issue by hewumars** [tensorflow/models#9364]
parameters：
--train_file_pattern=/home/data_ssd/hewu/vehicle_reid/train*
--validation_file_pattern=/home/data_ssd/hewu/vehicle_reid/validation*
--dataset_version=gld_v2_clean
--logdir=hz_training_2
--batch_size=256
--max_iters=150000
--initial_lr=0.01
--delg_global_features

hardware: 2080Ti GPU x8

commit-id: 85f5292

![image](https://user-images.githubusercontent.com/7970022/95697791-74c7fa80-0c72-11eb-9ae7-f275faa1aa0d.png)

![image](https://user-images.githubusercontent.com/7970022/95697803-7ee9f900-0c72-11eb-98b6-ed98b71b359d.png)

training accuracy  is always close to 0.
@dan-anghel @andrefaraujo

**Comment by dan-anghel** [comment]
Do you mind trying with a smaller initial learning rate, like --initial_lr=0.005, for 10k - 20k steps?

**Comment by andrefaraujo** [comment]
I can also see that you are setting `dataset_version ` but at the same time using a `vehicle_reid ` dataset? I wonder if this will lead to wrong number of classes or some issues with the training / accuracy computation.

At the same time, the descriptor loss does seem to be decreasing (although too slowly), so it's weird that the train_accuracy/desc would be zero.

**Comment by hewumars** [comment]
The dataset  is 46802 class , and the training accuracy can reach 0.7 in the previous version,but accuracy is low in tset dataset .after attn and desc loss were merged,training accuracy seem to be a problem.
